Debugging document is missing information

### .NET version

```
.NET SDK:
 Version:           9.0.100-rc.1.24452.12
 Commit:            81a714c6d3
 Workload version:  9.0.100-manifests.67cd1eb6
 MSBuild version:   17.12.0-preview-24422-09+d17ec720d
```

### Did it work in .NET Framework?

Yes

### Did it work in any of the earlier releases of .NET Core or .NET 5+?

_No response_

### Issue description

The [Debugging](https://github.com/dotnet/winforms/blob/main/docs/debugging.md) document is missing some information necessary to debug Windows Forms binaries. Specifically, for technique <span>#</span>2 to work, the following references must also be added.

```
<Reference Include="[Drive]:[Path-to-repo]\winforms\artifacts\bin\System.Windows.Forms\Debug\net9.0\System.Drawing.Common.dll" />
<Reference Include="[Drive]:[Path-to-repo]\winforms\artifacts\bin\System.Windows.Forms\Debug\net9.0\System.Private.Windows.Core.dll" />
```

### Steps to reproduce

Follow the steps for debugging Windows Forms binaries using technique <span>#</span>2 in the [Debugging](https://github.com/dotnet/winforms/blob/main/docs/debugging.md) document, build the solution, which results in error CS1705.
`Assembly 'System.Windows.Forms' with identity 'System.Windows.Forms, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089' uses 'System.Private.Windows.Core, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089' which has a higher version than referenced assembly 'System.Private.Windows.Core' with identity 'System.Private.Windows.Core, Version=9.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'`

--- a/docs/debugging.md
+++ b/docs/debugging.md
@@ -32,6 +32,8 @@ Add references to the binary(-ies) to your project ported to .NET. For example, 
 
 ```xml
 <ItemGroup>
+    <Reference Include="[Drive]:[Path-to-repo]\winforms\artifacts\bin\System.Windows.Forms\Debug\net7.0\System.Drawing.Common.dll" />
+    <Reference Include="[Drive]:[Path-to-repo]\winforms\artifacts\bin\System.Windows.Forms\Debug\net7.0\System.Private.Windows.Core.dll" />
     <Reference Include="[Drive]:[Path-to-repo]\winforms\artifacts\bin\System.Windows.Forms\Debug\net7.0\System.Windows.Forms.dll" />
     <Reference Include="[Drive]:[Path-to-repo]\winforms\artifacts\bin\System.Windows.Forms.Primitives\Debug\net7.0\System.Windows.Forms.Primitives.dll" />
     <!-- Optionally you may need designer -->

